README.md

```diff
@@ -1,3 +1,12 @@
 # NG
 
-Next Generation JavaScript IO Platform
+Forum for discussions about what the next generation of Node.js might look like.
+
+This repository is strictly for discussions about what people think Node.js might look like in the future.
+
+The Node.js project's governing bodies (CTC and TSC) tend to stay out of these discussions and allow those
+passionate about discussing these topics to enjoy using this forum.
+
+No particular thread is endorsed by the Node.js project as being part of any roadmap. If a discussion
+reaches a point of maturity it is expected to end up as a new module, an 
```

Comment:
Nit: `If a discussion reaches a point of maturity, then it is expected...`

(comma and `then` for clarity)